acceptance/tests/options/no_cache_should_not_refresh_cached_facts.rb

```diff
@@ -18,7 +18,12 @@
 
   config = <<-FILE
 cli : { debug : true }
-facts : { ttls : [ { "#{cached_fact_name}" : 30 minutes } ] }
+facts : {
+  blocklist : [ "az_metadata" ],
```

Comment:
I would prefer to update the test assertion rather than changing the preconditions here